wbooks/src/app/components/AppNavigator/config.tsx

```diff
@@ -2,20 +2,31 @@ import React from 'react';
 import { Image } from 'react-native';
 import backButton from '@assets/ic_back.png';
 import headerBackground from '@assets/bc_nav_bar.png';
+import SearchBar from '@components/SearchBar';
 import colors from '@constants/colors';
 
 import styles from './styles';
 
-const customHeader = {
+const sharedHeaderProperties = {
   headerStyle: {
     height: 90
   },
-  headerTintColor: colors.white,
-  headerBackImage: () => <Image style={styles.backButton} resizeMode="cover" source={backButton} />,
   headerBackground: () => <Image style={styles.headerImage} resizeMode="stretch" source={headerBackground} />,
```

Comment:
Why these changes?

Reply:
I needed a solution to have some common header elements that could be shared in order to obtain 2 different headers (with some similarities). After reading all your comments I realized the child screens will inherit them regardless.